spec/index.bs

```diff
@@ -10805,7 +10805,7 @@ dictionary GPURenderPassColorAttachment {
 dictionary GPURenderPassDepthStencilAttachment {
     required GPUTextureView view;
 
-    float depthClearValue = 0;
+    required float depthClearValue = 0;
```

Comment:
Thanks, yeah I had overlooked it.

Related: https://chromium-review.googlesource.com/c/chromium/src/+/4270352/comment/42756d87_2eb6d9cb/

I want to see how the spec decides.

Reply:
The depth/stencil LoadOp and StoreOp already have this issue. They're only required if the view has a depth or stencil aspect. To handle that we [validate them explicitly in the spec](https://gpuweb.github.io/gpuweb/#abstract-opdef-gpurenderpassdepthstencilattachment-gpurenderpassdepthstencilattachment-valid-usage) and a validation error is raised if they aren't set when required.

The most consistent thing to do here, then, would be to not use the IDL `required` keyword for the clear value and add additional validation to the previously linked section that raises a validation error if `depthClearValue` is not set AND view has a depth aspect.

That'll be a bit of a pain for the native implementations, since they'll have to pass something like `NaN` in to represent "not set", but it's workable.

Reply:
Thanks, updated.